lib/src/migrators/namespace.dart

```diff
@@ -83,7 +83,7 @@ class _NamespaceMigrationVisitor extends MigrationVisitor {
   @override
   void beforePatch(Stylesheet node) {
     // Pass each `@use` rule through the renamer.
-    var newNamespaces = <String, Set<UseRule>>{};
+    Map<String?, Set<UseRule>> newNamespaces = <String, Set<UseRule>>{};
```

Comment:
This looks like a bogus retype from the migrator. I tink the `rule.namespace == null` check below should be made into an intemediate variable, at which point this can genuinely have non-nullable keys.